### DISCUSSION TITLE
Guest Windows 2000 setup crashes

### DISCUSSION THREAD

**Discussion by WRadde** [utmapp/UTM#6142] [Windows]
[config.plist.pdf](https://github.com/utmapp/UTM/files/13639084/config.plist.pdf)

Windows 2000 sp0,sp1, sp4 all setup crashes or boot loop right after you put cd key and set network setting on utm version 4.4.4 (92) Mac Mini 2023 Apple M2 pro Sonoma 14.1.2
no crash report

**Comment by amcchord** [comment]
This is a race condition in the windows 2000 installer. You can fix by changing the cpu to qemu32,level=1 you can do that by adding lines to the Qemu config in settings. More info here: https://rubenerd.com/windows-2000-in-qemu-on-an-m1-mac/

**Reply by michalzablocki319** [reply]
Yeah, but there doesn't seem to be a way to change it in utm. How exactly can I get around it? Is there a way to edit the qemu config in utm?
